tests/common/devices.py

```diff
@@ -553,6 +553,19 @@ def get_version(self):
         """
         output = dut.command("sonic-cfggen -y /etc/sonic/sonic_version.yml -v build_version")
         return output["stdout_lines"][0].strip()
+    
+    def get_dut_iface_mac(self, iface_name):
+        """
+        Gets the MAC address of specified interface.
+
+        Returns:
+            str: The MAC address of the specified interface, or None if it is not found.
+        """
+        for iface, iface_info in self.setup()['ansible_facts'].items():
```

Comment:
I think that's a good idea but I'm thinking I will save that for a follow-up PR since the change also needed to be vetted for the existing `test_drop_counters.py` script.